static/modules/entity.js

```diff
@@ -100,6 +93,27 @@ export class Entity {
   /** @type {boolean} */
   pausable = true;
 
+  /**
+   * Determines what type of collision will be generated when getCollisionShape
+   * is called and collisionShape is undefined.
+   */
+  /** @type {"Box"|"Circle"|"undefined"} */
```

Comment:
Calling it "undefined" might cause some headaches later when we're trying to see if collisionType is actually "undefined" or `undefined` being coerced into "undefined". Can we pick a different name for this even if it's "not defined"?